### PR TITLE
release 2.4.1

### PR DESCRIPTION
**Pull request by gouthamve**
We have an enhancement in there because we don't have a release branch in `prometheus/tsdb`. Will be fixed for the 2.5 release. Not sure if we should mention that though.

depends on #4631

### CODE REVIEW

**Comment by brancz** [comment]
I have little time this week to look at anything, can we just make sure that the tsdb metrics are still correct, as I'm unsure whether we've cleaned up everything in prometheus/prometheus after deciding to revert one of the PRs @beorn7 did against prometheus/tsdb.

**Comment by beorn7** [comment]
The changes in prometheus/prometheus were never merged. We should be good metrics-wise.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 2.4.1 / 2018-09-19
+
+* [ENHANCEMENT] New TSDB metrics prometheus/tsdb#375 prometheus/tsdb#363
+* [BUGFIX] Render UI correctly for windows #4616
```
Comment (simonpasquier) [inline CHANGELOG.md]: s/windows/Windows/

Comment (gouthamve) [reply CHANGELOG.md]: Fixed, thanks!

**Pull request merged**